studio/schemas/schemaTypes/slug.ts

```diff
@@ -51,6 +52,13 @@ function createSlugField(source: string) {
           "Slug can only consist of latin letters, digits, hyphen (-), underscore (_), full stop (.) and tilde (~)"
         );
       }),
+    readOnly: (ctx) => {
+      /*
+        make slugs read-only after initial publish
+        to avoid breaking shared links
+       */
```

Comment:
maybe add that these can be changed in the redirect? or some documentation for how slugs and redirects are related?

Reply:
That feature is not available yet though, maybe add it in the redirects PR?

Reply:
Yes, will you make a note of it so not to forget?